Assets/Scripts/Object/AbstractObject.cs

```diff
@@ -1,36 +1,36 @@
-﻿using System;
-using UnityEngine;
+﻿using UnityEngine;
 using Utils;
 using Random = UnityEngine.Random;
 
-namespace Object
-{
-    public abstract class AbstractObject : MonoBehaviour
-    {
+namespace Object {
+    public abstract class AbstractObject : MonoBehaviour {
         private const float defaultRotationSpeed = 42F;
-        private const double BH_radius = 1;
+        private const float blackHoleRadius = 1.26F;
+
+        private bool mainMode;
 
         [SerializeField] private float minMoveSpeed;
         [SerializeField] private float maxMoveSpeed;
-        [SerializeField] private float minRotSpeed;
-        [SerializeField] private float maxRotSpeed;
-        [SerializeField] private AudioSource onDestroySound;
-
         private float moveSpeed;
         private Vector3 targetPosition;
+
+        [SerializeField] private float minRotSpeed;
+        [SerializeField] private float maxRotSpeed;
         private float rotSpeed;
         private Vector3 rotDirection;
-        // Reference to Spawner.cs
-        private Spawner spawner;
 
+        private bool sucked = false;
+        
+        [SerializeField] private AudioSource onDestroySound;
+
+        public void Start() {
+            mainMode = BlackHole.GetInstance() != null;
```

Comment:
![ask](https://img.shields.io/badge/review-ask-9b71ae.svg) 
What `mainMode` used for? Do you envision implementing a start screen or something in the future development?

Reply:
for now it's just useful for test scenes (because sometime there's no blackhole in test scenes), but yeah maybe it could also be useful in the future